### ISSUE TITLE
isSearchable can not be turned on again after turning it off

### ISSUE THREAD

**Issue by Tharit** [rob-balfre/svelte-select#196]
When setting "isSearchable" to false and then back to true again the "readonly" property is never removed;
this means the select is stuck in readonly mode.

Seems like a possible fix would be to set "_inputAttributes.readonly" to undefined so that the property is removed again;
```
if (!isSearchable) {
  _inputAttributes.readonly = true;
} else {
  _inputAttributes.readonly = undefined;
}
````

**Comment by rob-balfre** [comment]
@Tharit thanks. Yup, can confirm thats a bug... https://svelte.dev/repl/a859c2ba7d1744af9c95037c48989193?version=3.12.1

**Comment by rob-balfre** [comment]
Fixed in v4.0.0-beta.10 

https://svelte.dev/repl/5b2f37e0446449e89a4db3c02bcf6f26?version=3.12.1
